lib/core-integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -3437,7 +3437,7 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
         let minUtxoWithAsset = minutxo +
                 -- The extra amount is dependent on the era:
                 if _mainEra ctx >= ApiBabbage
-                then 176_710
+                then 193_950
```

Comment:
In future, perhaps it would be better to record the **_total_** as a "golden" constant, rather than trying to record lots of different constants and them adding them up.

Alternatively, since we already have good coverage of minimum UTxO values within property and golden tests, we could consider just removing these constants from our integration test suite (or deferring to a well-tested API function that returns the expected constant).

Reply:
Yeah, I've wondered if we can't do anything better with all of these fragile hard-coded coin expectations too... Though we could compare with the underlying implementation ("defer to a well-tested API function that returns the expected minimum"), I suspect the golden aspect of them _also_ has some value, and would be nice to preserve...